### PR TITLE
Create Interaction design page “typography”

### PR DESCRIPTION
**Pull request by mingoland**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/interaction-design/typography.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- Font size, line height, line length
+- Font weight
+- Typeface
+
+You may wonder why we discuss these three elements together, while others are mentioned separately. This is because font size, line height, and line length are interrelated and adjusting one will require reevaluating the balance with the others. They cannot be isolated when considering accessibility. And this time we will look at body text first, and then heading text after.
+
+## Body text
+
+### Font Size
+There have been various studies on the ideal font size for web content, and it is suggested to use 18-22 px for desktop and 16-18 px for mobile. This range is large enough to be readable in most situations. The GOV.UK Design System, a widely used front-end framework for public sector services, uses 19px for large screens, which is within this range. The final font size may also depend on the typeface chosen, but this recommendation should be suitable for most common web fonts.
```
Comment (yndajas) [inline src/interaction-design/typography.md:15]: A few instances of headings with space before the paragraph - would be good to include a space between them, but we can fix this later if it's not obvious how to do it in the CMS

File: src/interaction-design/typography.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
```
Comment (yndajas) [inline src/interaction-design/typography.md:17]: Likewise a few instances of double spaces that would be good to reduce to single spaces, but we can fix it later if it's not easy to spot in the CMS

File: src/interaction-design/typography.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+permalink: /:path/:basename/
+---
+There are several typography elements that can enhance the accessibility of web content. These elements include:
+- Font size, line height, line length
+- Font weight
+- Typeface
+
+You may wonder why we discuss these three elements together, while others are mentioned separately. This is because font size, line height, and line length are interrelated and adjusting one will require reevaluating the balance with the others. They cannot be isolated when considering accessibility. And this time we will look at body text first, and then heading text after.
+
+## Body text
```
Comment (yndajas) [inline src/interaction-design/typography.md:12]: Is it worth adding `To be added.` here like you have for font weight and typeface?

Comment (yndajas) [inline src/interaction-design/typography.md:12]: There's a bit of inconsistency in casing for headings: sentence or title case. The playbook defaults to sentence case, so maybe we should here, but that might be more for Calum to decide

File: src/interaction-design/typography.md (hunk 1 above)
Comment (yndajas) [inline src/interaction-design/typography.md:15]: Here you have `18-22 px`, `16-18 px` and `19px`, and elsewhere no space between the numbers and the unit. Perhaps change to `18-22px` and `16-18px` for consistency?

**Review by yndajas — COMMENTED** [review]
Will leave the proper content review to Calum, but just a few things I spotted in terms of formatting/consistency

File: src/interaction-design/typography.md (hunk 1 above)
Comment (mingoland) [reply src/interaction-design/typography.md:15]: @yndajas thanks for spotting all of them! Let me spend some time on it again.

**Review by calumryan — APPROVED** [review]

**Pull request merged**